src/main/java/uk/gov/companieshouse/officerfiling/api/validation/OfficerAppointmentValidator.java

```diff
@@ -67,6 +67,8 @@ public ApiErrors validate(HttpServletRequest request, OfficerFilingDto dto, Tran
         // Perform validation
         validateCompanyNotDissolved(request, errorList, companyProfile.get());
         validateAllowedCompanyType(request, errorList, companyProfile.get());
+        validateAppointmentPastOrPresent(request, errorList, dto);
```

Comment:
To keep things consistent, I'd call these form the validateRequiredDtoFields method when appointment date is being checked.

Reply:
Anona has pointed out that in TM01 validation we only check length and characters in the requiredDtoFields check. The implemented way fits with this